microsoft-edge/visual-studio-code/debugger-for-edge.md

```diff
@@ -1,14 +1,14 @@
 ---
-description: How to debug Microsoft Edge from Visual Studio Code
-title: Debug Microsoft Edge from Visual Studio Code
+description: How to get in touch with the Debugger for Microsoft Edge from Visual Studio Code Extension team
+title: Get in touch with the Debugger for Microsoft Edge Visual Studio Code Extension team
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 07/26/2021
+ms.date: 11/05/2021
 ms.topic: article
 ms.prod: microsoft-edge
 keywords: microsoft edge, web development, f12 tools, devtools, vs code, visual studio code, debugger
 ---
-# Debug Microsoft Edge from Visual Studio Code
+# Get in touch with the Debugger for Microsoft Edge Visual Studio Code Extension team
```

Comment:
The article isn't about getting in touch.  The article is a high-level overview of this extension, with a link.

Reply:
oudated